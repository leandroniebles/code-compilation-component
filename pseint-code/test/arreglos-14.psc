funcion foo(a)
  a(1)<-5;
finfuncion
proceso main
  n<-1;
  foo(n);
finproceso