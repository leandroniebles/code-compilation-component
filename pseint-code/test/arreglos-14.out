=== Line 2: ExeError 201
